doc/manual/plugins.rst

```diff
@@ -46,7 +50,6 @@ Global plugins
    plugins_global/errors
    plugins_global/rc
    plugins_global/wcsmatch
-   plugins_global/changehistory
```

Comment:
@pllim, once these are removed, we no longer actually have a section of the manual devoted to each plugin?